### PR TITLE
Carbonmark data token details pages

### LINKED ISSUE #1548: [carbon dashboard] Token Details Pages
- [x] Token Details page
  - [x] State of Digital Carbon Detail Page
  - [x] Breakdown Detail Page
  - [x] Volume Over Time Detail Page
  - [x] Distribution of Vintage Start Dates Detail Page
  - [x] Distribution of Methodologies Detail Page
  - [x] Distribution of Projects Detail Page

### PR DESCRIPTION
**Pull request by biwano**
## Description

Token details pages

## Related Ticket

Resolves #1548

## Changes

| Before  | After  |
|---------|--------|
|img here |img here|

## Checklist

- [ ] I have run `npm run build-all` without errors
- [ ] I have formatted JS and TS files with `npm run format-all`

### CODE REVIEW

File: carbonmark-data/components/Graphics/c3t.png
[the remark this answers is not in the document]
Comment (biwano) [reply carbonmark-data/components/Graphics/c3t.png:1]: Cannot find it. Can you create a Tech debt ticket about that?

File: carbonmark-data/components/pages/PageWithTabs/index.tsx [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+type TabParam = {
+  key: string;
+  label: string;
+  optionsList?: Array<Options<string>>;
+  content?: ReactNode;
+  contents?: Record<string, React.ReactNode>;
+};
+export type TabParams = Array<TabParam>;
+
+function TypedTabPanel(props: {
```
[the remark this answers is not in the document]
Comment (biwano) [reply carbonmark-data/components/pages/PageWithTabs/index.tsx]: Done as part of 52e72588a047b526a85a76c26d26793d2a74eede (missed atomic commit)

File: carbonmark-data/components/pages/PageWithTabs/index.tsx [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+  }, [optionKeys]);
+
+  // Handle Options change
+  function onOptionChange(tabIndex: number, widgetIndex: number) {
+    return (value: Key) => {
+      const newOptionKeys = [...optionKeys];
+      newOptionKeys[tabIndex][widgetIndex] = value;
+      setOptionKeys(newOptionKeys);
+    };
+  }
```
[the remark this answers is not in the document]
Comment (biwano) [reply carbonmark-data/components/pages/PageWithTabs/index.tsx:92]: I am not usre I am getting it. 
- The onOptionChange function is part of the PageWithTabs component
- The OptionSwitcher is used in both che ChartCard component and PageWithTabs component.

[the remark this answers is not in the document]
Comment (sprrwhwk) [reply carbonmark-data/components/pages/PageWithTabs/index.tsx:92]: Ok got it, my mistake 😬

File: carbonmark-data/components/Graphics/c3t.png
[the remark this answers is not in the document]
Comment (biwano) [reply carbonmark-data/components/Graphics/c3t.png:1]: Haaa! Yeah I understand now, we should do it. But please make a ticket, I do not want to take risks messing with other modules within this PR.

[the remark this answers is not in the document]
Comment (biwano) [reply carbonmark-data/components/Graphics/c3t.png:1]: And hold on 10 minutes I am changing the imports

[the remark this answers is not in the document]
Comment (biwano) [reply carbonmark-data/components/Graphics/c3t.png:1]: Done in 61b8e40f

**Pull request merged**